extensions/wikia/Email/Controller/CommentController.class.php

```diff
@@ -78,17 +78,17 @@ public function body() {
 			'editorUserName' => $this->getCurrentUserName(),
 			'editorAvatarURL' => $this->getCurrentAvatarURL(),
 			'summary' => $this->getSummary(),
-			'details' => $this->getDetails(),
```

Comment:
Actually this line change has nothing to do right now.
It was a problem when we had variable called ```title``` because there is some strange code that is modifying title parameter in a controller. After we renamed it everything should work ok